src/Servers/Kestrel/Core/src/KestrelServerLimits.cs

```diff
@@ -42,18 +42,27 @@ public class KestrelServerLimits
     private long? _maxConcurrentConnections;
     private long? _maxConcurrentUpgradedConnections;
 
+
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved
     /// <summary>
     /// Gets or sets the maximum size of the response buffer before write
     /// calls begin to block or return tasks that don't complete until the
     /// buffer size drops below the configured limit.
     /// Defaults to 65,536 bytes (64 KB).
+    /// <para>
+    /// For limits to work correctly, this needs to be consistent with
+    /// <code>MaxWriteBufferSize</code> in the transport options
+    /// (e.g. <see cref="Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportOptions.MaxWriteBufferSize"/>
```

Comment:
@Rick-Anderson Is this a reasonable way to link to a member that isn't visible from this location?

Reply:
I think it needs to start with P: for property.  I don’t know why it wouldn’t be visible, but if it isn’t then it should just be a `href `link.  @gewarren  is the expert.

Reply:
I think @Rick-Anderson is correct. See for example https://github.com/dotnet/runtime/blob/b41b63973bf49ad7a2d0d2ef37f08cf5df622ea1/src/libraries/System.Private.CoreLib/src/System/IO/RandomAccess.cs#L103.

Reply:
Oh, I see, you use the cref output syntax because the compiler won't be able to generate it for a symbol it can't resolve - cute.

@Rick-Anderson It's not visible _in code_ because it's internal to a different assembly.  I'm not sure what the visibility rules are for the generated doc page.